src/load/azext_load/data_plane/load_test/help.py

```diff
@@ -118,6 +118,28 @@
         az load test convert-to-jmx --load-test-resource sample-alt-resource --resource-group sample-rg --test-id sample-existing-test-id
 """
 
+helps[
+    "load test set-baseline"
+] = """
+type: command
+short-summary: Set a baseline test run for the test.
+examples:
+    - name: Set baseline test run.
+      text: |
+        az load test set-baseline --load-test-resource sample-alt-resource --resource-group sample-rg --test-id sample-existing-test-id --test-run-id sample-associated-test-run-id
+"""
+
+helps[
+    "load test compare-to-baseline"
+] = """
+type: command
+short-summary: Compare the sampler statistics of recent test runs with the baseline test run.
+examples:
+    - name: Compare recent test runs to baseline.
```

Comment:
@saloniagrawal1997 please review short-summary and name

Reply:
Compare the sampler statistics from recent test runs with those of the baseline test run.